### ISSUE TITLE
Unable to send the domain_hint with Blazor web assembly (using MSAL JS) 

### ISSUE THREAD

**Issue by singhnishant40** [dotnet/sdk#13297]
Issue reference: https://stackoverflow.com/questions/63605653/is-there-a-way-to-supply-a-domain-hint-for-single-sign-on-using-msal-net-on-a-bl

**Comment by umlaufa1** [comment]
FYI, domain_hint is a AuthenticationParameter of MSAL, which does not appear to be supported/exposed by the Microsoft.Authentication.WebAssembly.Msal package

Here are related issues for other Authentication parameters:
extraQueryParameter: #25391  (https://github.com/dotnet/aspnetcore/issues/25391)
loginHint: #19877 (https://github.com/dotnet/aspnetcore/issues/19925)

**Comment by szalapski** [comment]
Yes, this is important to us.  Users get a suboptimal experience without it.

**Comment by szalapski** [comment]
Would you be willing to prioritize it somewhat?  It definitely affects UX and so should be considered.

**Comment by umlaufa1** [comment]
I am just an early adoptor looking to use Blazor for a project in my business. You independently ran into this issue at the same time I did. The lack of Domain Hint and other MSAL parameters is a big drawback because it prohibits a rich login UI (which is one of the first things users see). Blazor is awsome and the dev community is great too. You may want to consider posting this issue under dotnet/aspnetcore where it might receive more attention.
